diff --git a/kernel/feature/selinux_hide.c b/kernel/feature/selinux_hide.c
--- a/kernel/feature/selinux_hide.c
+++ b/kernel/feature/selinux_hide.c
@@ -474,8 +474,7 @@ static void type_attribute_bounds_av(struct policydb *policydb,
     avd->allowed &= ~masked;
 
     /* audit masked permissions */
-    if (security_dump_masked_av_fn)
-        security_dump_masked_av_fn(policydb, scontext, tcontext, tclass, masked, "bounds");
+    security_dump_masked_av_fn(policydb, scontext, tcontext, tclass, masked, "bounds");
 }
 
 /*
@@ -791,11 +790,7 @@ static void security_compute_av(struct selinux_state *state, u32 ssid, u32 tsid,
         goto out;
     }
 
-    if (context_struct_compute_av_fn) {
-        context_struct_compute_av_fn(policydb, scontext, tcontext, tclass, avd, NULL);
-    } else {
-        context_struct_compute_av(policydb, scontext, tcontext, tclass, avd, NULL);
-    }
+    context_struct_compute_av_fn(policydb, scontext, tcontext, tclass, avd, NULL);
 out:
     return;
 allow:
